spec/requests/providers_spec.rb

```diff
@@ -1116,4 +1116,29 @@ def gen_import_request
       end
     end
   end
+
+  context "OPTIONS /api/providers" do
+    it "returns options for all providers when no query" do
+      api_basic_authorize
```

Comment:
Now that #8 has been merged you can remove this and subsequent calls to `api_basic_authorize`